src/utils/myAccountUtils.ts

```diff
@@ -0,0 +1,30 @@
+import type { Patron, SierraPatron } from "../types/myAccountTypes"
+
+export const notificationPreferenceMap = {
+  z: "Email",
+  p: "Phone",
+  m: "Mobile",
+}
+
+export const notificationPreferenceTuples = [
```

Comment:
Perhaps not likely but if a new notification preference is added, you'd have to update two variables.

Try using `notificationPreferenceMap` as the "source" and create `notificationPreferenceTuples` based on the source. That way, if there is a new preference in the future, you'd only have to update one variable.

Reply:
updated!